### PR TITLE
Add envinfo to gatsby-cli, issue template

### PR DESCRIPTION
**Pull request by tabrindle**
Add `gatsby info --clipboard` returns something like this:

Note: npmPackages and npmGlobalPackages will return all gatsby packages matching glob "gatsby*"
```
  System:
    OS: macOS High Sierra 10.13
    CPU: x64 Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
    Shell: 5.4.2 - /usr/local/bin/zsh
  Binaries:
    Node: 8.11.0 - ~/.nvm/versions/node/v8.11.0/bin/node
    Yarn: 1.5.1 - ~/.yarn/bin/yarn
    npm: 5.6.0 - ~/.nvm/versions/node/v8.11.0/bin/npm
  Browsers:
    Chrome: 67.0.3396.62
    Firefox: 59.0.2
    Safari: 11.0
  npmPackages:
    gatsby: 1.9.261
  npmGlobalPackages:
    gatsby-cli: 1.1.52
```

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-glamor* failed.

Built with commit b7733fd94da33cd73a6f89f5528ba91071e24576

https://app.netlify.com/sites/using-glamor/deploys/5b0ffda90733d5472cce8974

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -30,12 +30,11 @@ What should happen?
 What happened.
 
 ### Environment
-* Gatsby version (`npm list gatsby`):
-* gatsby-cli version (`gatsby --version`):
-* Node.js version:
-* Operating System:
+
+<!-- Required. Run `gatsby info --clipboard` in your gatsby project directory and paste its contents here. -->
```
Comment (pieh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe mention to update global `gatsby-cli` if command doesn't exist?

Comment (tabrindle) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Great point, but brings a bit of a catch 22 - for anyone to include this info, they have to be on the latest gatsby-cli, whether or not that's where the bug is or not. I know some projects like React Native prefer it this way. Might we include something like an npx command for the latest?

Comment (KyleAMathews) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I think adding a note saying this command was added in a recent version of gatsby-cli and they might need to upgrade to the latest to get it should be fine.

Also given the rapid pace of new people trying Gatsby, it won't be long before the majority of people using Gatsby have this.

Comment (tabrindle) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done.

**Comment by m-allanson** [comment]
I thought this had already been merged and found myself wanting to use it today. Thanks again @tabrindle!

**Comment by jlengstorf** [comment]
Hiya @tabrindle! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**